docs/sources/operations/loki-canary/_index.md

```diff
@@ -259,16 +259,23 @@ spec:
 If the other options are not sufficient for your use case, you can compile
 `loki-canary` yourself:
 
-```bash
-# clone the source tree
-$ git clone https://github.com/grafana/loki
+1. Clone the source tree.
 
-# build the binary
-$ make loki-canary
+    ```bash
+    $ git clone https://github.com/grafana/loki
+    ```
 
-# (optionally build the container image)
-$ make loki-canary-image
-```
+1. Build the binary.
```

Comment:
I don't think the numbering really matters once it renders
just an fyi, ignore if this is intentional :)

Reply:
It's actually a best practice to do the numbering this way.

https://grafana.com/docs/writers-toolkit/write/markdown-guide/#numbered-lists